.travis.yml

```diff
@@ -15,6 +15,10 @@ services:
 
 before_script:
   - psql -c "CREATE DATABASE testing_db;" -U postgres
+  - psql -c "ALTER ROLE postgres PASSWORD 'postgres';" -U postgres
 
 script:
   - npm test
+after_success: 
+  - npm run coverage 
+  - npm run undo:migration
```

Comment:
Add newline here.

Reply:
I have added a new line at the end of the file. You can only see it when you try to edit the file though. Kindly review